### PR TITLE
Fixed edits not showing save button

### PR DESCRIPTION
**Pull request by RoyalSix**
This change is [Reviewable](https://reviewable.io/reviews/unfoldingword/markdown-translatable/35)

### CODE REVIEW

**Comment by RoyalSix** [comment]
Dependency of https://github.com/unfoldingWord/tc-create-app/issues/321

File: src/core/markdown-converter.ts [hunk 1]
```diff
@@ -55,7 +55,7 @@ export const markdownToHtml = ({ markdown, filters = [] }) => {
   let _markdown = (markdown || '');
 
   // Make "easy" blockquote:
-  _markdown = markdown.replace(/\n\>/g, '  \n\>');
```
Comment (ancientTexts-net) [inline src/core/markdown-converter.ts:58]: Pretty sure this was working before the \\ change here

**Comment by ancientTexts-net** [comment]
___
*[src/core/markdown-converter.ts, line 58 at r1](https://reviewable.io/reviews/unfoldingword/markdown-translatable/35#-MKetR25B6xJY1heYm_c:-MKetR25B6xJY1heYm_d:bghm5vs) ([raw file](https://github.com/unfoldingword/markdown-translatable/blob/7b6320a79b469e6d733648c0f2f39660893b1f19/src/core/markdown-converter.ts#L58)):*
> ```XML
> 
>   // Make "easy" blockquote:
>   _markdown = markdown.replace(/\n\>/g, '  \n\>');
> ```

Pretty sure this was working without the \\ change here ?

**Review by RoyalSix — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/unfoldingword/markdown-translatable/35)* status: 0 of 3 files reviewed, 1 unresolved discussion (waiting on @ancientTexts-net)

___
*[src/core/markdown-converter.ts, line 58 at r1](https://reviewable.io/reviews/unfoldingword/markdown-translatable/35#-MKetR25B6xJY1heYm_c:-MKf5bJTAJovA23taFu3:b59ofd8) ([raw file](https://github.com/unfoldingword/markdown-translatable/blob/7b6320a79b469e6d733648c0f2f39660893b1f19/src/core/markdown-converter.ts#L58)):*
Ahh ok eslint was giving me an error because that character does not need to be escaped. I went the other route of actually including a backslash in the replace. 
in short 
`
>
`.match(/\n\>/g)

is the same as

`
>
`.match(/\n>/g)


@ancienttexts-net

**Comment by ancientTexts-net** [comment]
Thank you @RoyalSix  , @elsylambert  :

https://deploy-preview-35--markdown-translatable.netlify.app/

As of now the only issues I see are that the FIRST time you edit the markdown document (with sections & blocks OFF), it sends cursor to the bottom of the file

**Comment by ancientTexts-net** [comment]
___
*[__tests__/mardown-converter/markdown-html.ts, line 24 at r3](https://reviewable.io/reviews/unfoldingword/markdown-translatable/35#-MLENVkU47z56olFRhbH:-MLENVkU47z56olFRhbI:bluuld9) ([raw file](https://github.com/unfoldingword/markdown-translatable/blob/d1569c487ad65811d14e856b65e0cea422dc13fc/__tests__/mardown-converter/markdown-html.ts#L24)):*
> ```XML
>     generateTest('bold_italic');
>   });
>   it(`convert occurrence note html to markdown`, () => {
> ```

Good! New unit test

**Comment by ancientTexts-net** [comment]
___
*[src/components/block-editable/BlockEditable.js, line 80 at r3](https://reviewable.io/reviews/unfoldingword/markdown-translatable/35#-MLEODXx3H2bbXtw8jG2:-MLEODXx3H2bbXtw8jG3:bg5z5xg) ([raw file](https://github.com/unfoldingword/markdown-translatable/blob/7b6320a79b469e6d733648c0f2f39660893b1f19/src/components/block-editable/BlockEditable.js#L80)):*
> ```JavaScript
>   }, [inputFilters, markdown, preview]);
> 
>   /** Only updating the markdown when the mode
> ```

We agreed to WATCH these changes for performance

**Comment by ancientTexts-net** [comment]
___
*[src/core/markdown-converter.ts, line 58 at r1](https://reviewable.io/reviews/unfoldingword/markdown-translatable/35#-MKetR25B6xJY1heYm_c:-MLEOQlH6zcHLPb2EdIp:b750ifa) ([raw file](https://github.com/unfoldingword/markdown-translatable/blob/7b6320a79b469e6d733648c0f2f39660893b1f19/src/core/markdown-converter.ts#L58)):*
<details><summary><i>Previously, RoyalSix (Sterling (Jay) Scott) wrote…</i></summary><blockquote>

Ahh ok eslint was giving me an error because that character does not need to be escaped. I went the other route of actually including a backslash in the replace. 
in short 
`
>
`.match(/\n\>/g)

is the same as

`
>
`.match(/\n>/g)


@ancienttexts-net 
</blockquote></details>

Thank you Jay

**Review by ancientTexts-net — COMMENTED** [review]
> Reviewed 2 of 3 files at r1, 12 of 12 files at r3.
> *[Reviewable](https://reviewable.io/reviews/unfoldingword/markdown-translatable/35)* status: all files reviewed, 3 unresolved discussions (waiting on @RoyalSix)
